edx_proctoring/templates/timed_exam/submitted.html

```diff
@@ -10,5 +10,10 @@
     {% blocktrans %}
       Your grade for this timed exam will be immediately available on the <a href="{{progress_page_url}}">Progress</a> page.
     {% endblocktrans %}
+    {% if has_due_date %}
+      {% blocktrans %}
+        Once the exam due date has passed, you will be able to once again see your submitted exam answers but not change them.
```

Comment:
"After time has run out for the exam, you can review the exam, but you cannot change your answers."